index.html

```diff
@@ -136,6 +136,7 @@
         <pre class="example highlight json" title="typical manifest">
 {
   "lang": "en",
+  "dir": "ltr",
```

Comment:
> On 22 Mar 2016, at 7:35 PM, Kenneth Rohde Christiansen <notifications@github.com> wrote:
> 
> In index.html:
> 
> > @@ -136,6 +136,7 @@
> >          <pre class="example highlight json" title="typical manifest">
> >  {
> >    "lang": "en",
> > +  "dir": "ltr",
> Why was dir chosen instead of direction or directionality? Is this a common abbreviation for this? It made me think of directory :)
> 
Similarly named attributes in the HTMLs and the XMLs. 


> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>